templates/post/post.css

```diff
@@ -0,0 +1,208 @@
+/* main post styling */
+.post {
+  background-color: var(--color-off-white);
+}
+.post main {
+  background-color: var(--color-white);
+  margin:0 auto;
+  max-width: 460px;
+}
+.post main .section > div {
+  padding:0;
+  margin:0;
+}
+
+/* hero restyles */
+.post main .hero picture {
+  z-index:1;
+}
+
+.post main .hero {
+  margin: 0 auto;
+  padding: 0;
+}
+
+/* Post Hero  */
+@media (min-width: 600px) {
+  .post main {
+    column-gap: 1.5em;
+    display: grid;
+    grid-template-areas:
+    "hero hero"
+    "sidebar content"
+    "sidebar related-content"
+    "sidebar post-cards";
+    grid-template-columns: minmax(200px, 300px) minmax(50%, auto);
+    grid-template-rows: auto;
+    margin: 2rem auto;
+    max-width: var(--section-max-width);
+  }
+
+  .post main .section > div.hero-wrapper {
+    margin:0;
+  }
+
+  .post main > div.hero-container {
+    grid-area: hero;
+    padding-bottom:4em;
+  }
+}
+
+
+/* grid post sections */
+.post .section:nth-of-type(2) {
+  grid-area: content;
+}
+
+.post .section:nth-of-type(2) img {
+  height:auto;
+  max-width:100%;
+}
+
+.post .section:nth-of-type(2) picture + em {
+  background-color: var(--color-gray-100);
+  display:block;
+  font-size: var(--body-font-size-s);
+  margin: 10px 0;
+  padding: 6px 12px;
+  position: relative;
+  width:100%;
+}
+
+.post .section:nth-of-type(2) h3 {
```

Comment:
similar here. need to make this selector more robust so it won't break if the bog post content has section breaks